conans/client/graph/compute_pid.py

```diff
@@ -63,6 +63,10 @@ def compute_package_id(node, new_config):
 
     run_validate_package_id(conanfile)
 
+    if (conanfile.info.settings_target):
```

Comment:
```suggestion
    if conanfile.info.settings_target:
```
Parenthesis unnecessary